### PR TITLE
Qiuyesuifeng/tiny fix and cleanup

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
@siddontang  PTAL

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: server/cluster.go [hunk 1]
```diff
@@ -333,8 +333,9 @@ func (s *Server) bootstrapCluster(clusterID uint64, req *pdpb.BootstrapRequest)
 	cancel()
 	if err != nil {
 		return errors.Trace(err)
-	} else if !resp.Succeeded {
-		return errors.Errorf("bootstrap cluster %d fail, we may not leader", clusterID)
+	}
+	if !resp.Succeeded {
```
Comment (siddontang) [inline server/cluster.go]: why move below?

Comment (qiuyesuifeng) [inline server/cluster.go]: Nothing much to do, only unify the code format.

File: server/tso.go [hunk 2]
```diff
@@ -31,9 +31,12 @@ func getTimestampPath(rootPath string) string {
 
 func (s *Server) loadTimestamp() (int64, error) {
 	data, err := getValue(s.client, getTimestampPath(s.cfg.RootPath))
-	if err != nil || data == nil {
+	if err != nil {
```
Comment (siddontang) [inline server/tso.go]: I think above code is better and more tiny and compact.

Comment (qiuyesuifeng) [inline server/tso.go]: Seems more short, but in TiDB project, we have always checked the error first, then the value. So i think it is better to keep the same with that.

Comment (siddontang) [inline server/tso.go]: yes, here is first check err first , if no error and data is nil, return directly. so it is ok here.

File: server/cluster.go (hunk 1 above)
Comment (siddontang) [inline server/cluster.go]: `if else if` is commonly used in go, I think it is ok, no need to write only if multi times.

**Comment by siddontang** [comment]
Please unify all pd sources later.

LGTM

**Pull request merged**